keyboards/idank/spankbd/keymaps/default/keymap.json

```diff
@@ -0,0 +1,125 @@
+{
+    "version": 1,
+    "notes": "Span default keymap",
+    "documentation": "\"This file is a QMK Configurator export. You can import this at <https://config.qmk.fm>. It can also be used directly with QMK's source code.\n\nTo setup your QMK environment check out the tutorial: <https://docs.qmk.fm/#/newbs>\n\nYou can convert this file to a keymap.c using this command: `qmk json2c {keymap}`\n\nYou can compile this keymap using this command: `qmk compile {keymap}`\"\n",
+    "keyboard": "idank/spankbd",
+    "keymap": "default",
+    "layout": "LAYOUT_split_3x5_3",
+    "layers": [
+        [
+          "KC_Q",
+          "KC_W",
+          "KC_E",
+          "KC_R",
+          "KC_T",
+          "KC_Y",
+          "KC_U",
+          "KC_I",
+          "KC_O",
+          "KC_P",
+          "LCTL_T(KC_A)",
+          "KC_S",
+          "KC_D",
+          "KC_F",
+          "KC_G",
+          "KC_H",
+          "KC_J",
+          "KC_K",
+          "KC_L",
+          "LCTL_T(KC_SCLN)",
+          "RSFT_T(KC_Z)",
+          "KC_X",
+          "KC_C",
+          "KC_V",
+          "KC_B",
+          "KC_N",
+          "KC_M",
+          "KC_COMM",
+          "KC_DOT",
+          "RSFT_T(KC_SLSH)",
+          "LGUI_T(KC_ESC)",
+          "LALT_T(KC_ENT)",
+          "LT(1,KC_SPC)",
+          "LT(2,KC_SPC)",
+          "KC_LSFT",
+          "RSFT_T(KC_TAB)"
+        ],
+        [
+          "KC_EXLM",
+          "KC_AT",
+          "KC_LCBR",
+          "KC_RCBR",
+          "KC_PIPE",
+          "KC_GRV",
+          "KC_TILD",
+          "KC_TRNS",
+          "KC_TRNS",
+          "KC_BSLS",
+          "KC_HASH",
+          "KC_DLR",
+          "KC_LPRN",
+          "KC_RPRN",
+          "KC_BTN2",
+          "KC_PPLS",
+          "KC_MINS",
+          "KC_SLSH",
+          "KC_ASTR",
+          "KC_QUOT",
+          "KC_PERC",
+          "KC_CIRC",
+          "KC_LBRC",
+          "KC_RBRC",
+          "KC_BTN1",
+          "KC_AMPR",
+          "KC_EQL",
+          "KC_COMM",
+          "KC_DOT",
+          "KC_MINS",
+          "KC_BSPC",
+          "KC_SCLN",
+          "KC_EQL",
```

Comment:
```suggestion
          "_______",
```

Reply:
was this intentional?

Reply:
> was this intentional?

A better question is, how would you hit this key normally, as you'd have to be holding down `LT(1, KC_SPC)` to get to this layer?  And with a lack of any other layer keycodes to get to this layer ... what's the point of having a keycode here, when the transparent key is a good placeholder to note that this spot is taken/unusable?  

Which is most likely the motivation behind the request.

Reply:
done